### PR TITLE
Support more complex rdfa aware nodespecs

### PR DESCRIPTION
**Pull request by piemonkey**
### Overview
- Biggest change here is to add a way to pass an array instead of single child node to `renderRdfaAware`.
- Renamed last trailing references to 'content' nodes to 'literal' nodes
- Improved some types and the name of a type (deprecated the old one to make it backwards compatible).

##### connected issues and PRs:
Will be needed for a PR in plugins, will link here when it's ready.

### Setup
N/A

### How to test/reproduce
It shouldn't add anything new, so test that everything works. The only new thing will only be used by the new plugins code.

### Challenges/uncertainties
N/A

### Checks PR readiness
- [x] npm lint
- [x] no new deprecations

### CODE REVIEW

File: app/styles/ember-rdfa-editor/_c-annotation.scss [hunk 1]
```diff
@@ -62,8 +62,6 @@ $say-annotation-background-color: var(--au-gray-100) !default;
   [typeof]:not([typeof='foaf:Document']),
   [data-type],
   .mark-highlight-manual {
-    border-bottom: 0.2rem dotted var(--au-gray-300);
```
Comment (elpoelma) [inline app/styles/ember-rdfa-editor/_c-annotation.scss:65]: What is the reason for the style changes?

**Review by elpoelma — APPROVED** [review]
Looks good overall!

**Review by abeforgit — APPROVED** [review]

File: app/styles/ember-rdfa-editor/_c-annotation.scss (hunk 1 above)
Comment (piemonkey) [reply app/styles/ember-rdfa-editor/_c-annotation.scss:65]: The dotted line is no longer necessary to show what is contained within rdfa elements, as we now have the boxes to show this. Having both looks very broken. I guess maybe we should toggle whether this is displayed based on whether we're using the rdfa-aware editor or not?

Comment (elpoelma) [reply app/styles/ember-rdfa-editor/_c-annotation.scss:65]: It should also be possible to control these styles using the `rdfa-annotations` class. Which you can control using `editorOptions.showRdfa`. I suspect we will also remove a lot of the css of these rdfa-annotations, as the blocks we have now are IMO a better solution.

Comment (elpoelma) [reply app/styles/ember-rdfa-editor/_c-annotation.scss:65]: I would drop the style change from this PR, we can still remove it later and you can turn it off by setting `editorOptions.showRdfa` to `false`.

Comment (piemonkey) [reply app/styles/ember-rdfa-editor/_c-annotation.scss]: I've reverted the css change and removed `showRdfa` from the editable nodes route.

**Pull request merged**